src/Bridge/Ollama/LlamaModelHandler.php

```diff
@@ -41,7 +42,15 @@ public function request(Model $model, object|array|string $input, array $options
 
     public function convert(ResponseInterface $response, array $options = []): LlmResponse
     {
-        $data = $response->toArray();
+        $data = $response->toArray(false);
```

Comment:
same here, this code silences http error that are not handled below.

```suggestion
        $data = $response->toArray();
```